### PR TITLE
Clark/add translation workflow readme

### PR DESCRIPTION
**Pull request by clarkmcadoo**
Adding an updated translation workflow readme

### CODE REVIEW

File: TRANSLATION_WORKFLOW.md [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+
+The workflows are:
+
+- queueing translation requests
+  - this workflow takes a PR, looks for translatable files, and queues them up for either human or machine translation
+- sending content for machine translation
+  - this workflow takes queued up machine translation requests, and submits them to the vendor
+- sending content for human translation
+  - this workflow takes queued up human translation requests, and submits them to the vendor
+- checking for finished translations, downloading content, cleaning up
```
Comment (sunnyzanchi) [inline TRANSLATION_WORKFLOW.md:73]: i would capitalize these workflow names to match how they are in the actions tab (and maybe bold/italicize) for clarity

File: TRANSLATION_WORKFLOW.md [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+
+### Manually triggering workflow
+
+There is an additional github action to be able to manually queue up specific files. This can be manually triggered in the Actions tab of github under the workflow `Manually Add Slugs to Translation Queue`. When clicking `Run workflow` you will be prompted to enter the slugs that you would like to queue up, separated by comma. An example input would be `src/content/docs/codestream/how-use-codestream/discuss-code.mdx,src/content/docs/infrastructure/host-integrations/host-integrations-list/nginx/nginx-integration.mdx`. You additionally can select to queue it up for machine translation and give a locale, such as `ja-JP, ko-KR`.
+
+### Summary
+
+- Executes: on every merge to the `main` branch`
```
Comment (sunnyzanchi) [inline TRANSLATION_WORKFLOW.md:90]: stray backtick

```suggestion
- Executes: on every merge to the `main` branch
```

File: TRANSLATION_WORKFLOW.md [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+- Can be manually triggered? `Yes`, when using manual version of workflow
+- Steps:
+  - Looks at the files that have been changed on the merge
+  - Will save the filename, locale and status to the Translation table if not already pending
+    - `project_id` set to either machine or human translation project
```
Comment (sunnyzanchi) [inline TRANSLATION_WORKFLOW.md:95]: would it be a good idea to mention what the machine and human translation project ids are here?

**Review by sunnyzanchi — CHANGES_REQUESTED** [review]

File: TRANSLATION_WORKFLOW.md (hunk 3 above)
Comment (clarkmcadoo) [reply TRANSLATION_WORKFLOW.md:95]: We've gotta store these in One Password cuz they're kinda 🔒

**Comment by clarkmcadoo** [comment]
Just going to merge this in for now, we can revisit when we start the refactor to this process

**Pull request merged**